apps/tenant-management-webapp/src/app/store/configuration/sagas.ts

```diff
@@ -118,6 +104,29 @@ export function* fetchConfigurationDefinitions(_action: FetchConfigurationDefini
   }
 }
 
+async function fetchConfigurationNamespace(urlData: UrlData, token: string) {
```

Comment:
This should work with both mult-page calls and single page calls.  There shouldn't be any need to have the if statement on line 149.  If not, it shouldn't be hard to make it work for both.